src/pages/HomePage.tsx

```diff
@@ -0,0 +1,15 @@
+import { Navigate, useParams } from 'react-router-dom';
+
+export const HomePage = () => {
+  const { home } = useParams();
+
+  if (home === 'home') {
+    return <Navigate to="/" replace />;
+  }
```

Comment:
The check for `home === 'home'` and subsequent redirect to '/' seems unnecessary. If the `home` parameter is not required for any specific logic, consider removing this condition to simplify the component.